wasm/bindings/service_bindings.cpp

```diff
@@ -69,10 +69,9 @@ EMSCRIPTEN_BINDINGS(translation_model) {
 }
 
 EMSCRIPTEN_BINDINGS(blocking_service_config) {
-  value_object<BlockingService::Config>("BlockingServiceConfig");
-  // .field("name", &BlockingService::Config::name")
-  // The above is a future hook. Note that more will come - for cache, for workspace-size or graph details  limits on
-  // aggregate-batching etc.
+  value_object<BlockingService::Config>("BlockingServiceConfig")
+      .field("cacheEnabled", &BlockingService::Config::cacheEnabled)
+      .field("cacheSize", &BlockingService::Config::cacheSize);
```

Comment:
Note that this is an API breaking change: if you don't provide the the new configuration options when instantiating the blockingservice, it will error out. It's a limitation of value-objects and emscripten.